draft-xu-risav.md

```diff
@@ -317,13 +318,23 @@ Static negotiation makes endpoints nearly stateless, which simplifies the provis
 
 # Security Consideration
 
-## Incremental benefit from partial deployment
+## Threat models
 
-RISAV provides significant security benefits even if it is only deployed by a fraction of all ASes.  This is particularly clear in the context of reflection attacks.  If two networks implement RISAV, no one in any other network can trigger a reflection attack between these two networks.  Thus, if X% of ASes (selected at random) implement RISAV, participating ASes should see an X% reduction in reflection attack traffic volume.
+In general, RISAV seeks to provide a strong defense against arbitrary active attackers who are external to the source and destination AS.  However, different RISAV modes and configurations offer different security properties.
+
+### Replay attacks
+
+In Transport Mode, off-path attackers cannot spoof the source IPs of a participating AS, but any attacker with access to valid traffic can replay it (from anywhere), potentially enabling DoS attacks by replaying expensive traffic (e.g. TCP SYNs, QUIC Initials).  ASes that wish to have replay defense, and are willing to pay the extra data-plane costs, should prefer tunnel mode.
```

Comment:
As I have known, the RFC 4301-4303 say both IPsec AH and ESP have a sequence number field while the AH maybe not check this field and the ESP must check this field. It's up to the IPsec receiver. Shall we require that the ASBR implemented RISAV must check this sequence number field no matter whether it is AH or ESP?

Reply:
The problem is that we are in "multi-sender" mode, sharing a single SA across many ASBRs.  Per [RFC 4302 Section 3.4.3](https://datatracker.ietf.org/doc/html/rfc4302#section-3.4.3), this makes the sequence number field unusable:

>    However, this standard specifies no mechanisms for providing anti-
>    replay for a multi-sender SA (unicast or multicast).  In the absence
>    of negotiation (or manual configuration) of an anti-replay mechanism
>    for such an SA, it is recommended that sender and receiver checking
>    of the Sequence Number for the SA be disabled

This is because the different senders will have different sequence numbers, but the receiver has no way to distinguish the different senders.  This is why I've removed the sequence number entirely from the RISAV-AH header.

In Tunnel Mode, the different ASBRs are instructed to use distinct source IPs for RISAV.  This allows the receiver to maintain separate sequence number tracking for each sender, so sequence numbers work correctly and can be used for anti-replay.